restcomm/restcomm.sms/src/main/java/org/restcomm/connect/sms/smpp/dlr/provider/TelestaxDlrParser.java

```diff
@@ -127,4 +165,21 @@ public Status parseRestcommStatus(String message) {
         return status;
     }
 
+    private Error parseRestcommErrorCode(String errCode) {
```

Comment:
Duplicated logic in ErrorCodeMapper, with difference that code is a String. Can't we maintain consistency in single place?

Reply:
not really duplicated.one maping is for DLR err codes,the other for submitResp.they are diff mapping as documented